src/apphosting/githubConnections.ts

```diff
@@ -98,39 +99,55 @@ export async function linkGitHubRepository(
   utils.logBullet(clc.bold(`${clc.yellow("===")} Import a GitHub repository`));
   // Fetch the sentinel Oauth connection first which is needed to create further GitHub connections.
   const oauthConn = await getOrCreateOauthConnection(projectId, location);
-  const existingConns = await listAppHostingConnections(projectId, location);
+  let installationId = await promptGitHubInstallation(projectId, location, oauthConn);
 
-  if (existingConns.length === 0) {
-    existingConns.push(
-      await createFullyInstalledConnection(projectId, location, generateConnectionId(), oauthConn),
+  while (installationId === ADD_ACCOUNT_CHOICE) {
+    utils.logBullet(
+      "Install the Firebase App Hosting GitHub app on a new account to enable access to those repositories",
```

Comment:
Maybe this is a marissa question (this working of "adding an account" is on the console too). "Account" really means user or organization. I wonder if users are aware that "account" can also be an organization?

Reply:
talked about this with @qtz1 offline